pact4s/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/provider/LeoProvider.scala

```diff
@@ -89,29 +67,7 @@ class LeoProvider extends AnyFlatSpec with BeforeAndAfterAll with PactVerifier {
     )
 
   private val providerStatesHandler: StateManagementFunction = StateManagementFunction {
```

Comment:
Move state management functionality to separate files depending on the area of the app. This file only cares about combining the other state management functions.